script/add-grammar

```diff
@@ -43,16 +43,24 @@ def log(msg)
   puts msg if $verbose
 end
 
-def command(*args)
+def command(*args, show_warnings: true)
   log "$ #{args.join(' ')}"
-  output, status = Open3.capture2e(*args)
-  if !status.success?
+  stdout, stderr, status = Open3.capture3(*args)
+  unless status.success?
+    output = stdout.strip + "\n" + stderr.strip
     output.each_line do |line|
       log "  > #{line}"
     end
     warn "Command failed. Aborting."
     exit 1
   end
+  unless stderr.empty?
+    if show_warnings
+      stderr.each_line do |line|
+        log "  > #{line}"
```

Comment:
Out-of-scope for this PR, but do you think `script/add-grammar` would be better off implemented as a shell-script? I'd be more than willing to rewrite it (complete with options handling and everything), but it wouldn't hurt to know your thoughts on the matter.

Most of this file is just shelling out to external commands, and then echoing the output, which… feels pretty silly now that I think about it. 😕

Reply:
Most definitely! This whole shell in Ruby was really annoying me.